apps/examples/webserver_test/webserver_test_main.c

```diff
@@ -0,0 +1,999 @@
+
+	cur = param->headers->head->next;
+	while (cur != param->headers->tail) {
+		dest = ws_test_strcpy(dest, cur->key, param);
+		if (dest == NULL) {
+			return WS_TEST_MSG_CONSTRUCT_ERR;
+		}
+		dest = ws_test_strcpy(dest, ": ", param);
+		if (dest == NULL) {
+			return WS_TEST_MSG_CONSTRUCT_ERR;
+		}
+		dest = ws_test_strcpy(dest, cur->value, param);
+		if (dest == NULL) {
+			return WS_TEST_MSG_CONSTRUCT_ERR;
+		}
+		dest = ws_test_strcpy(dest, HTTPCRNL, param);
+		if (dest == NULL) {
+			return WS_TEST_MSG_CONSTRUCT_ERR;
+		}
+		cur = cur->next;
+	}
+	dest = ws_test_strcpy(dest, HTTPCRNL, param);
+	if (dest == NULL) {
+		return WS_TEST_MSG_CONSTRUCT_ERR;
+	}
+
+	/*----------------------------- Adding Entity for POST & PUT -------------------------------------*/
+
+	if (param->method == WS_TEST_MODE_POST || param->method == WS_TEST_MODE_PUT) {
+		/* content length */
+		if (!param->encoding) {
+			dest = ws_test_strcpy(dest, (char *)param->entity, param);
+			if (dest == NULL) {
+				return WS_TEST_MSG_CONSTRUCT_ERR;
+			}
+		} else { /* chunked param->encoding */
+			post_len = strlen(param->entity);
+			if (post_len > param->buflen) {
+				dest = ws_test_chunksize(dest, param->buflen, param);
+				if (dest == NULL) {
+					return WS_TEST_MSG_CONSTRUCT_ERR;
+				}
+				dest = ws_test_strcpy(dest, HTTPCRNL, param);
+				if (dest == NULL) {
+					return WS_TEST_MSG_CONSTRUCT_ERR;
+				}
+				dest = ws_test_strlencpy(dest, (char *)param->entity, param->buflen, param);
+				if (dest == NULL) {
+					return WS_TEST_MSG_CONSTRUCT_ERR;
+				}
+				dest = ws_test_strcpy(dest, HTTPCRNL, param);
+				if (dest == NULL) {
+					return WS_TEST_MSG_CONSTRUCT_ERR;
+				}
+			} else {
+				temp = dest;
+				dest = ws_test_chunksize(dest, post_len, param);
+				if (dest == NULL) {
+					return WS_TEST_MSG_CONSTRUCT_ERR;
+				}
+				dest = ws_test_strcpy(dest, HTTPCRNL, param);
+				if (dest == NULL) {
+					return WS_TEST_MSG_CONSTRUCT_ERR;
+				}
+				dest = ws_test_strcpy(dest, (char *)param->entity, param);
+				if (dest == NULL) {
+					return WS_TEST_MSG_CONSTRUCT_ERR;
+				}
+				dest = ws_test_strcpy(dest, HTTPCRNL, param);
+				if (dest == NULL) {
+					return WS_TEST_MSG_CONSTRUCT_ERR;
+				}
+				g_normal_clen = dest - temp;
+				dest = ws_test_strlencpy(dest, "0", 1, param);
+				if (dest == NULL) {
+					return WS_TEST_MSG_CONSTRUCT_ERR;
+				}
+				dest = ws_test_strcpy(dest, HTTPCRNL, param);
+				if (dest == NULL) {
+					return WS_TEST_MSG_CONSTRUCT_ERR;
+				}
+				if (param->trailer) {
+					dest = ws_test_strcpy(dest, EXPIRES, param);
+					if (dest == NULL) {
+						return WS_TEST_MSG_CONSTRUCT_ERR;
+					}
+				}
+				dest = ws_test_strcpy(dest, HTTPCRNL, param);
+				if (dest == NULL) {
+					return WS_TEST_MSG_CONSTRUCT_ERR;
+				}
+				g_last_clen = dest - temp;
+			}
+		}
+	}
+	FUNC_EX;
+	return dest - param->buffer;
+}
+
+static int ws_test_socket_connect(struct ws_test_s *ws)
+{
+	FUNC_EN;
+	int sockfd, ret;
```

Comment:
TizenRT coding rule prohibit this. please modify it like following and initialize value too.
```
int sockfd = 0;
int ret = 0;
```